### ISSUE TITLE
Meta ticket - Regroupement de facteurs

### ISSUE THREAD

**Issue by streino** [ecolabdata/ecospheres#145]
### User story
En tant que créateur d’un bouquet de données possédant un grand nombre de facteurs, je souhaite pouvoir regrouper mes facteurs afin de pouvoir les présenter de manière plus lisible.

### To do : 

- [x] #428
- [x] #429
- [x] #430
- [x] #431
- [x] #432
- [x] #435
- [x] #438
- [x] #433

Hors périmètre lot 1 : 
- [x] #434
- [x] #440 

Maquette : 
https://www.figma.com/design/goZNoxBnPfhh8awXjnDzY6/Maquettes-2024?node-id=612-10648&t=04c60gCKmAzutM1C-4
Attention aux boutons -->  pas de changement de design, on continue à utiliser la librairie datagouv component

**Comment by Thesauruv** [comment]
On peut déjà réordonner les jeux de données, mais effectivement ça gagnerait en lisibilité de pouvoir faire des sous-groupes. to do sprint 4

**Comment by streino** [comment]
A articuler avec #228

**Comment by DanFluture275** [comment]
Priorité haute : à faire via un système de tags et possibilité de les filtrer

**Comment by ghost** [comment]
Dernière version disponible : https://www.figma.com/design/goZNoxBnPfhh8awXjnDzY6/Maquettes-2024?node-id=612-10648&node-type=section&t=ehOaMxusg7gDV6Zw-11

**Comment by DanFluture275** [comment]
**Retours suite à démo du 21/11**

- Dans la page bouquet : 
  -  ne pas afficher le label "Sans regroupement", et afficher directement les JDD à la fin (qui seront donc indentés par rapport aux regroupements). 
  - Ajouter un compteur du nombre de JDD présent dans le regroupement : label "X jeux de données". Design sous forme de tag ?
- Dans le formulaire :
  - Changer le titre : "Inclure dans un regroupement (facultatif)"
  - Changer la petite description en dessous : "Sélectionner un regroupement existant ou en créer un nouveau."
  - Positionnement du champs dans le formulaire d'ajout depuis un bouquet : mettre à la toute fin
  - Positionnement du champs dans le formulaire d'ajout depuis un JDD : le passer en deuxième position
 
- Attention a traiter également le cas fonctionnel pour en pas afficher le "Sans regroupement" pour les bouquets sans regroupements --> du coup sera géré aussi via le traitement du premier point

**Comment by abulte** [comment]
>   * ne pas afficher le label "Sans regroupement", et afficher directement les JDD à la fin (qui seront donc indentés par rapport aux regroupements).

Pas chaud pour faire ça sans maquette et sans autre retour. Il y a une version qui marche et tient debout niveau UI. On pourra itérer par la suite si besoin.

>   * Ajouter un compteur du nombre de JDD présent dans le regroupement : label "X jeux de données". Design sous forme de tag ?

Je pense qu'on peut attendre aussi, niveau design ce n'est pas forcément évident.

>   * Changer le titre : "Inclure dans un regroupement (facultatif)"

Attention à la cohérence avec le reste des champs : pourquoi un verbe ici et pas ailleurs ?

D'une manière générale, c'est très bien de prendre en compte les retours mais ce n'est pas forcément une bonne idée d'actionner les choses dans l'urgence et sans plus de réflexion.

**Comment by DanFluture275** [comment]
oui, oui, j'étais dans l'idée d'en reparler lundi, c'était pour garder une trace des échanges
